tests/deepsparse/evaluation/test_utils.py

```diff
@@ -47,12 +47,12 @@ def torch_target():
 
 def test_initialize_model_from_target_pipeline_onnx(pipeline_target):
     model = text_generation_model_from_target(pipeline_target, "onnxruntime")
-    assert model.engine_type == "onnxruntime"
+    assert model.ops.get("single_engine")._engine_type == "onnxruntime"
```

Comment:
oh man, is this the simplest way to fetch this attribute?

Reply:
for now. pipelines list all ops, including the engine ops, in a dict so we'll have to reference the key for the time being